### PR TITLE
added service

### PR DESCRIPTION
**Pull request by mohitkaul17**
please review my pull request for the feature a

### CODE REVIEW

File: src/main/java/com/pardeep/Service.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.pardeep;
+
+public class Service {
```
Comment (psdhanesar) [inline src/main/java/com/pardeep/Service.java:3]: add interface for the service

File: src/main/java/com/pardeep/Service.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    public boolean addAccount() {
+        System.out.println("The Account has been added");
+        return true;
+    }
+
```
Comment (psdhanesar) [inline src/main/java/com/pardeep/Service.java:8]: this is good

**Review by psdhanesar — CHANGES_REQUESTED** [review]

**Review by balkiratsingh — APPROVED** [review]

File: src/main/java/com/pardeep/ServiceInterf.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package com.pardeep;
+
+public interface ServiceInterf {
```
Comment (psdhanesar) [inline src/main/java/com/pardeep/ServiceInterf.java:3]: good job

**Review by psdhanesar — APPROVED** [review]

File: src/main/java/com/pardeep/Service.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    public boolean isActive() {
+        System.out.println("The account is inactive");
+        return false;
+    }
+
```
Comment (balkiratsingh) [inline src/main/java/com/pardeep/Service.java:13]: perfect

**Pull request merged**